pkg/manifests/manifests_test.go

```diff
@@ -828,6 +828,13 @@ func TestPrometheusOperatorAdmissionWebhookConfiguration(t *testing.T) {
 	c, err := NewConfigFromString(`prometheusOperator:
```

Comment:
```suggestion
	c, err := NewConfigFromString(`prometheusOperatorAdmissionWebhook:
```
should be updated

Reply:
and I see func
`func TestPrometheusOperatorAdmissionWebhookOwnConfiguration(t *testing.T) {`
in
https://github.com/openshift/cluster-monitoring-operator/blob/7770ea5ee04d5f90392287300306f927d300ad95/pkg/manifests/manifests_test.go#L907
maybe we could add the test under this func?

Reply:
yes, better

Reply:
> IMHO, it should be updated

is this a bug? There is no changes in this func and change it in this PR could be confused

Reply:
please ignore the comment, since the case is put under TestPrometheusOperatorAdmissionWebhookOwnConfiguration func now